.changeset/quick-toes-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
```

Comment:
Also checking that this is a patch, so only waiting for the feature to merge but not for the next minor release? If so, please help me remember when the corresponding docs PR should merge, since it's not on the normal release schedule!  :smile:  https://github.com/withastro/docs/pull/5410

Reply:
Once the PR is merged, this new thing will go out in the **next** release (regardless of its entity). 

> remember when the corresponding docs PR should merge

Ideally, we should merge it before the next release. Although we don't have a schedule for that, I think it's also okay if the docs PR is released 1/2 day after.